Dockerfile

```diff
@@ -48,8 +42,6 @@ RUN curl -sSL "http://neuro.debian.net/lists/$( lsb_release -c | cut -f2 ).us-ca
 RUN apt-get update && \
     DEBIAN_FRONTEND=noninteractive \
     apt-get install -y --no-install-recommends \
-                    fsl-core=5.0.9-5~nd18.04+1 \
-                    fsl-mni152-templates=5.0.7-2 \
```

Comment:
I assume we just use templateflow here anyway?

Reply:
If we're using this template it will be a problem anyways (it's in the wrong orientation for xcpd). So if we remove it and see a problem it's a good thing.